force-app/main/default/classes/SOQL.cls

```diff
@@ -63,9 +78,11 @@ public with sharing class SOQL {
      * @return A list of Opportunities with important opportunities fields
      * Update the corresponding test class and testGetClosedWonOpportunityCountByAccount method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
+    public static List<Opportunity> getClosedWonOpportunityCountByAccount(Id accountId) {
     //  The body of the method
-    //}
+        List<Opportunity> accountOpportunities = [SELECT Id, Name, Description, Amount, StageName, AccountId, CloseDate from Opportunity where AccountId = :accountId and StageName = 'Closed Won'];
+        return accountOpportunities;
```

Comment:
This is the first place where you'll want to start thinking about query formatting. As your queries get bigger and more complex, you will want to start formatting them vertically so they are easier to read.
```suggestion
        List<Opportunity> opps = [
                SELECT 
                        Id,
                        Name,
                        StageName,
                        AccountId,
                        CloseDate,
                        Amount 
                FROM Opportunity 
                WHERE AccountId = :accId 
                AND StageName = 'Closed Won'
        ];
```